config/index.js

```diff
@@ -815,6 +815,14 @@ var conf = convict({
         env: 'RECOVERY_CODE_COUNT'
       }
     }
+  },
+  changePassword: {
+    minLastAuth: {
+      doc: 'Minimum amount of time that the session has been last verified (milliseconds) 5 minute default',
```

Comment:
Naming nits: I'm pretty sure this is a maximum not a minimum, and it's about authentication not verification.